pkg/types/interval.go

```diff
@@ -25,8 +25,41 @@ func (i Interval) Seconds() int {
 	return m
 }
 
+// specially handled, for better precision
```

Comment:
go doc comment always starts with the function name, for here it is `Milliseconds {verb} .... `

Reply:
check this page https://tip.golang.org/doc/comment

Reply:
done